src/main/react-webapp/src/components/Home.js

```diff
@@ -1,12 +1,36 @@
 import React from 'react';
-import {NavLink} from 'react-router-dom';
+import {NavLink, useHistory} from 'react-router-dom';
 import '../css/home.css'
-import drugs from '../images/drugs.png'
+import drugsImg from '../images/drugs.png'
 import map from '../images/map.png'
 import sideEffects from '../images/sideEffects.png'
 
 const Home = () => {
 
+    let history = useHistory();
+
+    const handleChange = e => {
+        console.log(e.target.value, e.target.selectedOptions[0].label)
+        const value = e.target.value
+        const label = e.target.selectedOptions[0].label
+        console.log(e.target)
```

Comment:
probably want to remove console.log and commented out lines if this is now working.